piccolo_api/media/s3.py

```diff
@@ -127,13 +144,24 @@ def store_file_sync(
         A sync wrapper around :meth:`store_file`.
         """
         file_key = self.generate_file_key(file_name=file_name, user=user)
-
+        extension = file_key.rsplit(".", 1)[-1]
         client = self.get_client()
+        metadata: t.Dict[str, t.Any] = {
+            "ACL": self.default_acl,
+            "ContentDisposition": "inline",
```

Comment:
Should we `ContentDisposition` configurable, by adding a `content_disposition` parameter?

Reply:
Yeah, we can do. No issue from my side.